src/Kernel32.Shared/Kernel32+CreateFileFlags.cs

```diff
@@ -146,13 +145,13 @@ public enum CreateFileFlags : uint
             SECURITY_SQOS_PRESENT = 0x00100000,
 
             /// <summary>Impersonates a client at the Anonymous impersonation level.</summary>
-            SECURITY_ANONYMOUS = SECURITY_IMPERSONATION_LEVEL.SecurityAnonymous << 16,
+            SECURITY_ANONYMOUS = 0 << 16,
 
             /// <summary>The security tracking mode is dynamic. If this flag is not specified, the security tracking mode is static.</summary>
             SECURITY_CONTEXT_TRACKING = 0x00040000,
 
             /// <summary>Impersonates a client at the Delegation impersonation level.</summary>
-            SECURITY_DELEGATION = SECURITY_IMPERSONATION_LEVEL.SecurityDelegation << 16,
+            SECURITY_DELEGATION = 3 << 16,
```

Comment:
I think we should mirror the header files. Not as a strict rule since omitting an enum when it can't be used in a store app seems reasonable. But certainly in cases like this where we're losing semantically rich information and hard-coding values as a result, I think the least bad option is to go ahead and define the constants/enums in Store if necessary so that other things can depend on them.